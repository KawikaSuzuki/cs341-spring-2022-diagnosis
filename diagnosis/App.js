import 'react-native-gesture-handler';
import React from 'react';
import { LogBox } from 'react-native';

//warning catchers
LogBox.ignoreLogs([
  "[react-native-gesture-handler] Seems like you\'re using an old API with gesture components, check out new Gestures system!",
  "[react-native-gesture-handler] Seems like you're using an old API with gesture components, check out new Gestures system!",
  "Overwriting fontFamily style attribute preprocessor"
]);


//screens import
import StackNavigator from './navigators/StackNavigator';

export default function App() {
  return <StackNavigator />;
}