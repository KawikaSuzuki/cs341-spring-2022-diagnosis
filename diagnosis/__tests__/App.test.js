import { render } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';

import App from '../App';

describe('<App />', () => {
    test('dummy test', () => {
        expect(4+7).toBe(11);
        // const tree = renderer.create(<App />).toJSON();
        // expect(tree).toMatchSnapshot();
    });
});