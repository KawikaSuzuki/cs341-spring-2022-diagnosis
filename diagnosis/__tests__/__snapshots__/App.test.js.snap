// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNSScreenStack
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RNSScreen
      forwardedRef={[Function]}
      onAppear={[Function]}
      onDisappear={[Function]}
      onDismissed={[Function]}
      onWillDisappear={[Function]}
      replaceAnimation="push"
      stackPresentation="push"
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <RNSScreenStackHeaderConfig
        backButtonInCustomView={false}
        backgroundColor="transparent"
        color="#FFFFFF"
        direction="ltr"
        disableBackButtonMenu={false}
        hidden={false}
        hideBackButton={false}
        hideShadow={true}
        largeTitleHideShadow={false}
        title=""
        titleColor="#FFFFFF"
        topInsetEnabled={false}
        translucent={true}
      />
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            Object {
              "backgroundColor": "rgb(242, 242, 242)",
            },
            undefined,
          ]
        }
      >
        <View
          onLayout={[Function]}
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <RCTScrollView>
            <View>
              <View
                accessible={true}
                className="sc-bdvvtL zxezy"
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
              >
                <View
                  className="sc-gsDKAQ rZFaE"
                >
                  <Image
                    className="sc-dkPtRN fQWDve"
                    resizeMode="contain"
                    source={1}
                  />
                  <Text
                    className="sc-hKwDye fAcRlN"
                  >
                    Account Login
                  </Text>
                  <View
                    className="sc-eCImPb kMRaWM"
                  >
                    <View>
                      <View
                        className="sc-iCfMLu eAWtPN"
                      >
                        <Text />
                      </View>
                      <Text
                        className="sc-gKclnd dsijCe"
                      >
                        Username
                      </Text>
                      <TextInput
                        allowFontScaling={true}
                        className="sc-jRQBWg fNWPqr"
                        onBlur={[Function]}
                        onChangeText={[Function]}
                        placeholder="username"
                        placeholderTextColor="#ff000067"
                        rejectResponderTermination={true}
                        underlineColorAndroid="transparent"
                        value=""
                      />
                    </View>
                    <View>
                      <View
                        className="sc-iCfMLu eAWtPN"
                      >
                        <Text />
                      </View>
                      <Text
                        className="sc-gKclnd dsijCe"
                      >
                        Password
                      </Text>
                      <TextInput
                        allowFontScaling={true}
                        className="sc-jRQBWg fNWPqr"
                        onBlur={[Function]}
                        onChangeText={[Function]}
                        placeholder="********"
                        placeholderTextColor="#ff000067"
                        rejectResponderTermination={true}
                        secureTextEntry={true}
                        underlineColorAndroid="transparent"
                        value=""
                      />
                      <View
                        accessible={true}
                        focusable={true}
                        onClick={[Function]}
                        onResponderGrant={[Function]}
                        onResponderMove={[Function]}
                        onResponderRelease={[Function]}
                        onResponderTerminate={[Function]}
                        onResponderTerminationRequest={[Function]}
                        onStartShouldSetResponder={[Function]}
                        style={
                          Object {
                            "opacity": 1,
                          }
                        }
                      >
                        <Text />
                      </View>
                    </View>
                    <Text>
                      

                    </Text>
                    <View
                      accessible={true}
                      focusable={true}
                      onClick={[Function]}
                      onResponderGrant={[Function]}
                      onResponderMove={[Function]}
                      onResponderRelease={[Function]}
                      onResponderTerminate={[Function]}
                      onResponderTerminationRequest={[Function]}
                      onStartShouldSetResponder={[Function]}
                      style={
                        Object {
                          "opacity": 1,
                        }
                      }
                    >
                      <Text
                        className="sc-jrQzAO dafyie"
                      >
                        Login
                      </Text>
                    </View>
                    <View
                      className="sc-kDTinF glXNyz"
                    />
                    <View
                      accessible={true}
                      focusable={true}
                      onClick={[Function]}
                      onResponderGrant={[Function]}
                      onResponderMove={[Function]}
                      onResponderRelease={[Function]}
                      onResponderTerminate={[Function]}
                      onResponderTerminationRequest={[Function]}
                      onStartShouldSetResponder={[Function]}
                      style={
                        Object {
                          "opacity": 1,
                        }
                      }
                    >
                      <Text />
                      <Text
                        className="sc-jrQzAO gGIeVH"
                        wordpress={true}
                      >
                        Sign in with Wordpress
                      </Text>
                    </View>
                    <View
                      className="sc-iqseJM DBbKg"
                    >
                      <Text
                        className="sc-crHmcD bLjgwf"
                      >
                        Don't have an account? 
                      </Text>
                      <View
                        accessible={true}
                        focusable={true}
                        onClick={[Function]}
                        onResponderGrant={[Function]}
                        onResponderMove={[Function]}
                        onResponderRelease={[Function]}
                        onResponderTerminate={[Function]}
                        onResponderTerminationRequest={[Function]}
                        onStartShouldSetResponder={[Function]}
                        style={
                          Object {
                            "opacity": 1,
                          }
                        }
                      >
                        <Text
                          className="sc-bqiRlB kAwtfN"
                        >
                          Signup
                        </Text>
                      </View>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </RCTScrollView>
        </View>
      </View>
    </RNSScreen>
  </RNSScreenStack>
</RNCSafeAreaProvider>
`;
